### DISCUSSION TITLE
Row reordering/drag and drop

### DISCUSSION THREAD

**Discussion by vbeffa** [Comcast/react-data-grid#3258] [Q&A]
Does row reordering work? I see it in the [demos](https://adazzle.github.io/react-data-grid/#/rows-reordering) and I see the [example code](https://github.com/adazzle/react-data-grid/blob/28a5c3727cd5fbe29e88fe2e138cddcf1799de17/website/demos/RowsReordering.tsx) for it, but I have not been able to get it working, and this [README entry](https://github.com/adazzle/react-data-grid/blob/3687bb3053ca1aafb90f9b5aae7257aa2c812a10/CHANGELOG.md?plain=1#L323) indicates support for it was removed.

The demo uses `DraggableRowRenderer` but I don't see that in the exported types, and if I copy [that component](https://github.com/adazzle/react-data-grid/blob/3d2b379e0d4062e76e9e52f4d9ab62d4128ace1d/website/demos/components/DraggableRowRenderer.tsx) into my codebase I run into other problems like not having the right type for `Row`. (The `Row` in `index.d.ts` is a one-line const, whereas the [`Row`](https://github.com/adazzle/react-data-grid/blob/270e9bf051d0217678a7da37b656fc733ef764b3/src/Row.tsx) used in `DraggableRowRenderer` is a component.)

**Comment by godon019** [comment]
Hey the example code seem to be quite old. with current version, you can see this is working well here
https://codesandbox.io/s/wild-rgb-dtz8gr
you can drag and drop the field under ID column
